!SESSION 2014-09-05 17:54:30.651 -----------------------------------------------
eclipse.buildId=3.6.0.201407100658
java.version=1.7.0_67
java.vendor=Oracle Corporation
BootLoader constants: OS=linux, ARCH=x86, WS=gtk, NL=pt_BR
Command-line arguments:  -os linux -ws gtk -arch x86

!ENTRY com.aptana.projects 2 0 2014-09-05 17:55:19.931
!MESSAGE Internal error: 
!STACK 1
org.eclipse.core.runtime.CoreException: 
	at com.aptana.projects.wizards.AbstractNewProjectWizard$1.run(AbstractNewProjectWizard.java:253)
	at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2345)
	at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2327)
	at com.aptana.projects.wizards.AbstractNewProjectWizard.doCreateProject(AbstractNewProjectWizard.java:321)
	at com.aptana.projects.wizards.AbstractNewProjectWizard$2.run(AbstractNewProjectWizard.java:374)
	at org.eclipse.jface.operation.ModalContext$ModalContextThread.run(ModalContext.java:121)
Caused by: org.eclipse.core.runtime.CoreException: fatal: repository '/home/joao/Projetos/instituto' does not exist
	at com.aptana.projects.wizards.AbstractNewProjectWizard.doCloneFromGit(AbstractNewProjectWizard.java:698)
	at com.aptana.ruby.ui.wizards.NewRubyProjectWizard.cloneFromGit(NewRubyProjectWizard.java:145)
	at com.aptana.projects.wizards.AbstractNewProjectWizard.createNewProject(AbstractNewProjectWizard.java:570)
	at org.radrails.rails.internal.ui.NewRailsProjectWizard.createNewProject(NewRailsProjectWizard.java:116)
	at com.aptana.projects.wizards.AbstractNewProjectWizard$1.run(AbstractNewProjectWizard.java:249)
	... 5 more
!SUBENTRY 1 com.aptana.projects 4 0 2014-09-05 17:55:19.932
!MESSAGE 
!STACK 1
org.eclipse.core.runtime.CoreException: fatal: repository '/home/joao/Projetos/instituto' does not exist
	at com.aptana.projects.wizards.AbstractNewProjectWizard.doCloneFromGit(AbstractNewProjectWizard.java:698)
	at com.aptana.ruby.ui.wizards.NewRubyProjectWizard.cloneFromGit(NewRubyProjectWizard.java:145)
	at com.aptana.projects.wizards.AbstractNewProjectWizard.createNewProject(AbstractNewProjectWizard.java:570)
	at org.radrails.rails.internal.ui.NewRailsProjectWizard.createNewProject(NewRailsProjectWizard.java:116)
	at com.aptana.projects.wizards.AbstractNewProjectWizard$1.run(AbstractNewProjectWizard.java:249)
	at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2345)
	at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2327)
	at com.aptana.projects.wizards.AbstractNewProjectWizard.doCreateProject(AbstractNewProjectWizard.java:321)
	at com.aptana.projects.wizards.AbstractNewProjectWizard$2.run(AbstractNewProjectWizard.java:374)
	at org.eclipse.jface.operation.ModalContext$ModalContextThread.run(ModalContext.java:121)
!SUBENTRY 2 com.aptana.core 4 0 2014-09-05 17:55:19.932
!MESSAGE fatal: repository '/home/joao/Projetos/instituto' does not exist

!ENTRY com.aptana.projects 2 0 2014-09-05 17:55:35.928
!MESSAGE Internal error: 
!STACK 1
org.eclipse.core.runtime.CoreException: 
	at com.aptana.projects.wizards.AbstractNewProjectWizard$1.run(AbstractNewProjectWizard.java:253)
	at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2345)
	at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2327)
	at com.aptana.projects.wizards.AbstractNewProjectWizard.doCreateProject(AbstractNewProjectWizard.java:321)
	at com.aptana.projects.wizards.AbstractNewProjectWizard$2.run(AbstractNewProjectWizard.java:374)
	at org.eclipse.jface.operation.ModalContext$ModalContextThread.run(ModalContext.java:121)
Caused by: org.eclipse.core.runtime.CoreException: fatal: repository '/home/joao/Projetos/instituto' does not exist
	at com.aptana.projects.wizards.AbstractNewProjectWizard.doCloneFromGit(AbstractNewProjectWizard.java:698)
	at com.aptana.ruby.ui.wizards.NewRubyProjectWizard.cloneFromGit(NewRubyProjectWizard.java:145)
	at com.aptana.projects.wizards.AbstractNewProjectWizard.createNewProject(AbstractNewProjectWizard.java:570)
	at org.radrails.rails.internal.ui.NewRailsProjectWizard.createNewProject(NewRailsProjectWizard.java:116)
	at com.aptana.projects.wizards.AbstractNewProjectWizard$1.run(AbstractNewProjectWizard.java:249)
	... 5 more
!SUBENTRY 1 com.aptana.projects 4 0 2014-09-05 17:55:35.929
!MESSAGE 
!STACK 1
org.eclipse.core.runtime.CoreException: fatal: repository '/home/joao/Projetos/instituto' does not exist
	at com.aptana.projects.wizards.AbstractNewProjectWizard.doCloneFromGit(AbstractNewProjectWizard.java:698)
	at com.aptana.ruby.ui.wizards.NewRubyProjectWizard.cloneFromGit(NewRubyProjectWizard.java:145)
	at com.aptana.projects.wizards.AbstractNewProjectWizard.createNewProject(AbstractNewProjectWizard.java:570)
	at org.radrails.rails.internal.ui.NewRailsProjectWizard.createNewProject(NewRailsProjectWizard.java:116)
	at com.aptana.projects.wizards.AbstractNewProjectWizard$1.run(AbstractNewProjectWizard.java:249)
	at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2345)
	at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2327)
	at com.aptana.projects.wizards.AbstractNewProjectWizard.doCreateProject(AbstractNewProjectWizard.java:321)
	at com.aptana.projects.wizards.AbstractNewProjectWizard$2.run(AbstractNewProjectWizard.java:374)
	at org.eclipse.jface.operation.ModalContext$ModalContextThread.run(ModalContext.java:121)
!SUBENTRY 2 com.aptana.core 4 0 2014-09-05 17:55:35.929
!MESSAGE fatal: repository '/home/joao/Projetos/instituto' does not exist

!ENTRY com.aptana.projects 2 0 2014-09-05 17:56:20.219
!MESSAGE Internal error: 
!STACK 1
org.eclipse.core.runtime.CoreException: 
	at com.aptana.projects.wizards.AbstractNewProjectWizard$1.run(AbstractNewProjectWizard.java:253)
	at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2345)
	at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2327)
	at com.aptana.projects.wizards.AbstractNewProjectWizard.doCreateProject(AbstractNewProjectWizard.java:321)
	at com.aptana.projects.wizards.AbstractNewProjectWizard$2.run(AbstractNewProjectWizard.java:374)
	at org.eclipse.jface.operation.ModalContext$ModalContextThread.run(ModalContext.java:121)
Caused by: org.eclipse.core.runtime.CoreException: fatal: repository '/home/joao/Projetos/instituto' does not exist
	at com.aptana.projects.wizards.AbstractNewProjectWizard.doCloneFromGit(AbstractNewProjectWizard.java:698)
	at com.aptana.ruby.ui.wizards.NewRubyProjectWizard.cloneFromGit(NewRubyProjectWizard.java:145)
	at com.aptana.projects.wizards.AbstractNewProjectWizard.createNewProject(AbstractNewProjectWizard.java:570)
	at org.radrails.rails.internal.ui.NewRailsProjectWizard.createNewProject(NewRailsProjectWizard.java:116)
	at com.aptana.projects.wizards.AbstractNewProjectWizard$1.run(AbstractNewProjectWizard.java:249)
	... 5 more
!SUBENTRY 1 com.aptana.projects 4 0 2014-09-05 17:56:20.219
!MESSAGE 
!STACK 1
org.eclipse.core.runtime.CoreException: fatal: repository '/home/joao/Projetos/instituto' does not exist
	at com.aptana.projects.wizards.AbstractNewProjectWizard.doCloneFromGit(AbstractNewProjectWizard.java:698)
	at com.aptana.ruby.ui.wizards.NewRubyProjectWizard.cloneFromGit(NewRubyProjectWizard.java:145)
	at com.aptana.projects.wizards.AbstractNewProjectWizard.createNewProject(AbstractNewProjectWizard.java:570)
	at org.radrails.rails.internal.ui.NewRailsProjectWizard.createNewProject(NewRailsProjectWizard.java:116)
	at com.aptana.projects.wizards.AbstractNewProjectWizard$1.run(AbstractNewProjectWizard.java:249)
	at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2345)
	at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2327)
	at com.aptana.projects.wizards.AbstractNewProjectWizard.doCreateProject(AbstractNewProjectWizard.java:321)
	at com.aptana.projects.wizards.AbstractNewProjectWizard$2.run(AbstractNewProjectWizard.java:374)
	at org.eclipse.jface.operation.ModalContext$ModalContextThread.run(ModalContext.java:121)
!SUBENTRY 2 com.aptana.core 4 0 2014-09-05 17:56:20.220
!MESSAGE fatal: repository '/home/joao/Projetos/instituto' does not exist

!ENTRY com.aptana.projects 2 0 2014-09-05 17:56:34.918
!MESSAGE Internal error: 
!STACK 1
org.eclipse.core.runtime.CoreException: 
	at com.aptana.projects.wizards.AbstractNewProjectWizard$1.run(AbstractNewProjectWizard.java:253)
	at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2345)
	at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2327)
	at com.aptana.projects.wizards.AbstractNewProjectWizard.doCreateProject(AbstractNewProjectWizard.java:321)
	at com.aptana.projects.wizards.AbstractNewProjectWizard$2.run(AbstractNewProjectWizard.java:374)
	at org.eclipse.jface.operation.ModalContext$ModalContextThread.run(ModalContext.java:121)
Caused by: org.eclipse.core.runtime.CoreException: fatal: repository '/home/joao/Projetos/instituto' does not exist
	at com.aptana.projects.wizards.AbstractNewProjectWizard.doCloneFromGit(AbstractNewProjectWizard.java:698)
	at com.aptana.ruby.ui.wizards.NewRubyProjectWizard.cloneFromGit(NewRubyProjectWizard.java:145)
	at com.aptana.projects.wizards.AbstractNewProjectWizard.createNewProject(AbstractNewProjectWizard.java:570)
	at org.radrails.rails.internal.ui.NewRailsProjectWizard.createNewProject(NewRailsProjectWizard.java:116)
	at com.aptana.projects.wizards.AbstractNewProjectWizard$1.run(AbstractNewProjectWizard.java:249)
	... 5 more
!SUBENTRY 1 com.aptana.projects 4 0 2014-09-05 17:56:34.919
!MESSAGE 
!STACK 1
org.eclipse.core.runtime.CoreException: fatal: repository '/home/joao/Projetos/instituto' does not exist
	at com.aptana.projects.wizards.AbstractNewProjectWizard.doCloneFromGit(AbstractNewProjectWizard.java:698)
	at com.aptana.ruby.ui.wizards.NewRubyProjectWizard.cloneFromGit(NewRubyProjectWizard.java:145)
	at com.aptana.projects.wizards.AbstractNewProjectWizard.createNewProject(AbstractNewProjectWizard.java:570)
	at org.radrails.rails.internal.ui.NewRailsProjectWizard.createNewProject(NewRailsProjectWizard.java:116)
	at com.aptana.projects.wizards.AbstractNewProjectWizard$1.run(AbstractNewProjectWizard.java:249)
	at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2345)
	at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2327)
	at com.aptana.projects.wizards.AbstractNewProjectWizard.doCreateProject(AbstractNewProjectWizard.java:321)
	at com.aptana.projects.wizards.AbstractNewProjectWizard$2.run(AbstractNewProjectWizard.java:374)
	at org.eclipse.jface.operation.ModalContext$ModalContextThread.run(ModalContext.java:121)
!SUBENTRY 2 com.aptana.core 4 0 2014-09-05 17:56:34.919
!MESSAGE fatal: repository '/home/joao/Projetos/instituto' does not exist

!ENTRY com.aptana.projects 2 0 2014-09-05 17:56:50.854
!MESSAGE Internal error: 
!STACK 1
org.eclipse.core.runtime.CoreException: 
	at com.aptana.projects.wizards.AbstractNewProjectWizard$1.run(AbstractNewProjectWizard.java:253)
	at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2345)
	at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2327)
	at com.aptana.projects.wizards.AbstractNewProjectWizard.doCreateProject(AbstractNewProjectWizard.java:321)
	at com.aptana.projects.wizards.AbstractNewProjectWizard$2.run(AbstractNewProjectWizard.java:374)
	at org.eclipse.jface.operation.ModalContext$ModalContextThread.run(ModalContext.java:121)
Caused by: org.eclipse.core.runtime.CoreException: fatal: repository '/home/joao/Projetos/instituto' does not exist
	at com.aptana.projects.wizards.AbstractNewProjectWizard.doCloneFromGit(AbstractNewProjectWizard.java:698)
	at com.aptana.ruby.ui.wizards.NewRubyProjectWizard.cloneFromGit(NewRubyProjectWizard.java:145)
	at com.aptana.projects.wizards.AbstractNewProjectWizard.createNewProject(AbstractNewProjectWizard.java:570)
	at org.radrails.rails.internal.ui.NewRailsProjectWizard.createNewProject(NewRailsProjectWizard.java:116)
	at com.aptana.projects.wizards.AbstractNewProjectWizard$1.run(AbstractNewProjectWizard.java:249)
	... 5 more
!SUBENTRY 1 com.aptana.projects 4 0 2014-09-05 17:56:50.855
!MESSAGE 
!STACK 1
org.eclipse.core.runtime.CoreException: fatal: repository '/home/joao/Projetos/instituto' does not exist
	at com.aptana.projects.wizards.AbstractNewProjectWizard.doCloneFromGit(AbstractNewProjectWizard.java:698)
	at com.aptana.ruby.ui.wizards.NewRubyProjectWizard.cloneFromGit(NewRubyProjectWizard.java:145)
	at com.aptana.projects.wizards.AbstractNewProjectWizard.createNewProject(AbstractNewProjectWizard.java:570)
	at org.radrails.rails.internal.ui.NewRailsProjectWizard.createNewProject(NewRailsProjectWizard.java:116)
	at com.aptana.projects.wizards.AbstractNewProjectWizard$1.run(AbstractNewProjectWizard.java:249)
	at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2345)
	at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2327)
	at com.aptana.projects.wizards.AbstractNewProjectWizard.doCreateProject(AbstractNewProjectWizard.java:321)
	at com.aptana.projects.wizards.AbstractNewProjectWizard$2.run(AbstractNewProjectWizard.java:374)
	at org.eclipse.jface.operation.ModalContext$ModalContextThread.run(ModalContext.java:121)
!SUBENTRY 2 com.aptana.core 4 0 2014-09-05 17:56:50.855
!MESSAGE fatal: repository '/home/joao/Projetos/instituto' does not exist
!SESSION 2014-09-06 11:17:04.976 -----------------------------------------------
eclipse.buildId=unknown
java.version=1.7.0_67
java.vendor=Oracle Corporation
BootLoader constants: OS=linux, ARCH=x86, WS=gtk, NL=pt_BR
Framework arguments:  -application com.aptana.ide.desktop.integration.Application
Command-line arguments:  -os linux -ws gtk -arch x86 -application com.aptana.ide.desktop.integration.Application

!ENTRY org.eclipse.equinox.ds 4 0 2014-09-06 11:17:06.153
!MESSAGE 
!STACK 0
org.osgi.framework.BundleException: The bundle could not be resolved. Reason: Missing Constraint: Import-Package: org.eclipse.equinox.internal.util.event; version="1.0.0"
	at org.eclipse.osgi.framework.internal.core.AbstractBundle.getResolverError(AbstractBundle.java:1313)
	at org.eclipse.osgi.framework.internal.core.AbstractBundle.getResolutionFailureException(AbstractBundle.java:1297)
	at org.eclipse.osgi.framework.internal.core.BundleHost.startWorker(BundleHost.java:309)
	at org.eclipse.osgi.framework.internal.core.AbstractBundle.resume(AbstractBundle.java:370)
	at org.eclipse.osgi.framework.internal.core.Framework.resumeBundle(Framework.java:1068)
	at org.eclipse.osgi.framework.internal.core.StartLevelManager.resumeBundles(StartLevelManager.java:557)
	at org.eclipse.osgi.framework.internal.core.StartLevelManager.incFWSL(StartLevelManager.java:464)
	at org.eclipse.osgi.framework.internal.core.StartLevelManager.doSetStartLevel(StartLevelManager.java:248)
	at org.eclipse.osgi.framework.internal.core.StartLevelManager.dispatchEvent(StartLevelManager.java:445)
	at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:227)
	at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.run(EventManager.java:337)

!ENTRY org.eclipse.equinox.p2.directorywatcher 4 0 2014-09-06 11:17:21.319
!MESSAGE The installable unit org.eclipse.core.net.linux.x86.translated_host_properties is missing the filename property.

!ENTRY org.eclipse.equinox.p2.directorywatcher 4 0 2014-09-06 11:17:21.319
!MESSAGE The installable unit org.eclipse.ecf.provider.filetransfer.httpclient.ssl.translated_host_properties is missing the filename property.

!ENTRY org.eclipse.equinox.p2.directorywatcher 4 0 2014-09-06 11:17:21.320
!MESSAGE The installable unit org.eclipse.ecf.ssl.translated_host_properties is missing the filename property.

!ENTRY org.eclipse.equinox.p2.directorywatcher 4 0 2014-09-06 11:17:21.320
!MESSAGE The installable unit org.eclipse.ui.workbench.compatibility.translated_host_properties is missing the filename property.

!ENTRY org.eclipse.equinox.p2.directorywatcher 4 0 2014-09-06 11:17:21.321
!MESSAGE The installable unit org.eclipse.swt.gtk.linux.x86.translated_host_properties is missing the filename property.

!ENTRY org.eclipse.equinox.p2.directorywatcher 4 0 2014-09-06 11:17:21.321
!MESSAGE The installable unit org.eclipse.equinox.launcher.gtk.linux.x86.translated_host_properties is missing the filename property.

!ENTRY org.eclipse.equinox.p2.directorywatcher 4 0 2014-09-06 11:17:21.527
!MESSAGE The installable unit org.eclipse.core.net.linux.x86.translated_host_properties is missing the filename property.

!ENTRY org.eclipse.equinox.p2.directorywatcher 4 0 2014-09-06 11:17:21.527
!MESSAGE The installable unit org.eclipse.ecf.provider.filetransfer.httpclient.ssl.translated_host_properties is missing the filename property.

!ENTRY org.eclipse.equinox.p2.directorywatcher 4 0 2014-09-06 11:17:21.528
!MESSAGE The installable unit org.eclipse.ecf.ssl.translated_host_properties is missing the filename property.

!ENTRY org.eclipse.equinox.p2.directorywatcher 4 0 2014-09-06 11:17:21.528
!MESSAGE The installable unit org.eclipse.ui.workbench.compatibility.translated_host_properties is missing the filename property.

!ENTRY org.eclipse.equinox.p2.directorywatcher 4 0 2014-09-06 11:17:21.528
!MESSAGE The installable unit org.eclipse.swt.gtk.linux.x86.translated_host_properties is missing the filename property.

!ENTRY org.eclipse.equinox.p2.directorywatcher 4 0 2014-09-06 11:17:21.528
!MESSAGE The installable unit org.eclipse.equinox.launcher.gtk.linux.x86.translated_host_properties is missing the filename property.

!ENTRY org.eclipse.equinox.p2.directorywatcher 4 0 2014-09-06 11:17:21.818
!MESSAGE The installable unit org.eclipse.core.net.linux.x86.translated_host_properties is missing the filename property.

!ENTRY org.eclipse.equinox.p2.directorywatcher 4 0 2014-09-06 11:17:21.818
!MESSAGE The installable unit org.eclipse.ecf.provider.filetransfer.httpclient.ssl.translated_host_properties is missing the filename property.

!ENTRY org.eclipse.equinox.p2.directorywatcher 4 0 2014-09-06 11:17:21.818
!MESSAGE The installable unit org.eclipse.ecf.ssl.translated_host_properties is missing the filename property.

!ENTRY org.eclipse.equinox.p2.directorywatcher 4 0 2014-09-06 11:17:21.818
!MESSAGE The installable unit org.eclipse.ui.workbench.compatibility.translated_host_properties is missing the filename property.

!ENTRY org.eclipse.equinox.p2.directorywatcher 4 0 2014-09-06 11:17:21.819
!MESSAGE The installable unit org.eclipse.swt.gtk.linux.x86.translated_host_properties is missing the filename property.

!ENTRY org.eclipse.equinox.p2.directorywatcher 4 0 2014-09-06 11:17:21.819
!MESSAGE The installable unit org.eclipse.equinox.launcher.gtk.linux.x86.translated_host_properties is missing the filename property.

!ENTRY org.eclipse.equinox.p2.reconciler.dropins 4 0 2014-09-06 11:17:22.074
!MESSAGE 
!STACK 0
org.osgi.framework.BundleException: Exception in org.eclipse.equinox.internal.p2.reconciler.dropins.Activator.start() of bundle org.eclipse.equinox.p2.reconciler.dropins.
	at org.eclipse.osgi.framework.internal.core.BundleContextImpl.startActivator(BundleContextImpl.java:806)
	at org.eclipse.osgi.framework.internal.core.BundleContextImpl.start(BundleContextImpl.java:755)
	at org.eclipse.osgi.framework.internal.core.BundleHost.startWorker(BundleHost.java:352)
	at org.eclipse.osgi.framework.internal.core.AbstractBundle.resume(AbstractBundle.java:370)
	at org.eclipse.osgi.framework.internal.core.Framework.resumeBundle(Framework.java:1068)
	at org.eclipse.osgi.framework.internal.core.StartLevelManager.resumeBundles(StartLevelManager.java:557)
	at org.eclipse.osgi.framework.internal.core.StartLevelManager.incFWSL(StartLevelManager.java:464)
	at org.eclipse.osgi.framework.internal.core.StartLevelManager.doSetStartLevel(StartLevelManager.java:248)
	at org.eclipse.osgi.framework.internal.core.StartLevelManager.dispatchEvent(StartLevelManager.java:445)
	at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:227)
	at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.run(EventManager.java:337)
Caused by: java.lang.IllegalStateException: Profile AptanaProfile is not current.
	at org.eclipse.equinox.internal.p2.engine.SimpleProfileRegistry.lockProfile(SimpleProfileRegistry.java:691)
	at org.eclipse.equinox.internal.provisional.p2.engine.Engine.perform(Engine.java:44)
	at org.eclipse.equinox.internal.p2.reconciler.dropins.ProfileSynchronizer.executePlan(ProfileSynchronizer.java:466)
	at org.eclipse.equinox.internal.p2.reconciler.dropins.ProfileSynchronizer.synchronize(ProfileSynchronizer.java:107)
	at org.eclipse.equinox.internal.p2.reconciler.dropins.Activator.synchronize(Activator.java:422)
	at org.eclipse.equinox.internal.p2.reconciler.dropins.Activator.start(Activator.java:171)
	at org.eclipse.osgi.framework.internal.core.BundleContextImpl$1.run(BundleContextImpl.java:783)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.eclipse.osgi.framework.internal.core.BundleContextImpl.startActivator(BundleContextImpl.java:774)
	... 10 more
Root exception:
java.lang.IllegalStateException: Profile AptanaProfile is not current.
	at org.eclipse.equinox.internal.p2.engine.SimpleProfileRegistry.lockProfile(SimpleProfileRegistry.java:691)
	at org.eclipse.equinox.internal.provisional.p2.engine.Engine.perform(Engine.java:44)
	at org.eclipse.equinox.internal.p2.reconciler.dropins.ProfileSynchronizer.executePlan(ProfileSynchronizer.java:466)
	at org.eclipse.equinox.internal.p2.reconciler.dropins.ProfileSynchronizer.synchronize(ProfileSynchronizer.java:107)
	at org.eclipse.equinox.internal.p2.reconciler.dropins.Activator.synchronize(Activator.java:422)
	at org.eclipse.equinox.internal.p2.reconciler.dropins.Activator.start(Activator.java:171)
	at org.eclipse.osgi.framework.internal.core.BundleContextImpl$1.run(BundleContextImpl.java:783)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.eclipse.osgi.framework.internal.core.BundleContextImpl.startActivator(BundleContextImpl.java:774)
	at org.eclipse.osgi.framework.internal.core.BundleContextImpl.start(BundleContextImpl.java:755)
	at org.eclipse.osgi.framework.internal.core.BundleHost.startWorker(BundleHost.java:352)
	at org.eclipse.osgi.framework.internal.core.AbstractBundle.resume(AbstractBundle.java:370)
	at org.eclipse.osgi.framework.internal.core.Framework.resumeBundle(Framework.java:1068)
	at org.eclipse.osgi.framework.internal.core.StartLevelManager.resumeBundles(StartLevelManager.java:557)
	at org.eclipse.osgi.framework.internal.core.StartLevelManager.incFWSL(StartLevelManager.java:464)
	at org.eclipse.osgi.framework.internal.core.StartLevelManager.doSetStartLevel(StartLevelManager.java:248)
	at org.eclipse.osgi.framework.internal.core.StartLevelManager.dispatchEvent(StartLevelManager.java:445)
	at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:227)
	at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.run(EventManager.java:337)

!ENTRY org.eclipse.core.resources 4 567 2014-09-06 11:36:03.381
!MESSAGE Workspace restored, but some problems occurred.
!SUBENTRY 1 org.eclipse.core.resources 4 567 2014-09-06 11:36:03.381
!MESSAGE Could not read metadata for 'Instituto'.
!STACK 1
org.eclipse.core.internal.resources.ResourceException: The project description file (.project) for 'Instituto' is missing.  This file contains important information about the project.  The project will not function properly until this file is restored.
	at org.eclipse.core.internal.localstore.FileSystemResourceManager.read(FileSystemResourceManager.java:714)
	at org.eclipse.core.internal.resources.SaveManager.restoreMetaInfo(SaveManager.java:797)
	at org.eclipse.core.internal.resources.SaveManager.restoreMetaInfo(SaveManager.java:777)
	at org.eclipse.core.internal.resources.SaveManager.restore(SaveManager.java:663)
	at org.eclipse.core.internal.resources.SaveManager.startup(SaveManager.java:1353)
	at org.eclipse.core.internal.resources.Workspace.startup(Workspace.java:1957)
	at org.eclipse.core.internal.resources.Workspace.open(Workspace.java:1716)
	at org.eclipse.core.resources.ResourcesPlugin.start(ResourcesPlugin.java:379)
	at org.eclipse.osgi.framework.internal.core.BundleContextImpl$1.run(BundleContextImpl.java:783)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.eclipse.osgi.framework.internal.core.BundleContextImpl.startActivator(BundleContextImpl.java:774)
	at org.eclipse.osgi.framework.internal.core.BundleContextImpl.start(BundleContextImpl.java:755)
	at org.eclipse.osgi.framework.internal.core.BundleHost.startWorker(BundleHost.java:352)
	at org.eclipse.osgi.framework.internal.core.AbstractBundle.start(AbstractBundle.java:280)
	at org.eclipse.osgi.framework.util.SecureAction.start(SecureAction.java:408)
	at org.eclipse.core.runtime.internal.adaptor.EclipseLazyStarter.postFindLocalClass(EclipseLazyStarter.java:111)
	at org.eclipse.osgi.baseadaptor.loader.ClasspathManager.findLocalClass(ClasspathManager.java:449)
	at org.eclipse.osgi.internal.baseadaptor.DefaultClassLoader.findLocalClass(DefaultClassLoader.java:211)
	at org.eclipse.osgi.internal.loader.BundleLoader.findLocalClass(BundleLoader.java:381)
	at org.eclipse.osgi.internal.loader.SingleSourcePackage.loadClass(SingleSourcePackage.java:33)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClassInternal(BundleLoader.java:454)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:410)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:398)
	at org.eclipse.osgi.internal.baseadaptor.DefaultClassLoader.loadClass(DefaultClassLoader.java:105)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:358)
	at com.aptana.radrails.rcp.IDEApplication.start(IDEApplication.java:122)
	at com.aptana.ide.desktop.integration.Application.start(Application.java:56)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:194)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:110)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:79)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:368)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:179)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:559)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:514)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1311)
!SUBENTRY 2 org.eclipse.core.resources 4 567 2014-09-06 11:36:03.382
!MESSAGE The project description file (.project) for 'Instituto' is missing.  This file contains important information about the project.  The project will not function properly until this file is restored.

!ENTRY org.eclipse.ui.workbench 2 0 2014-09-06 11:36:06.433
!MESSAGE A handler conflict occurred.  This may disable some commands.
!SUBENTRY 1 org.eclipse.ui.workbench 2 0 2014-09-06 11:36:06.433
!MESSAGE Conflict for 'AUTOGEN:::org.rubypeople.rdt.refactoring.refactoringContext/org.rubypeople.rdt.refactoring.action.RenameAction':
HandlerActivation(commandId=AUTOGEN:::org.rubypeople.rdt.refactoring.refactoringContext/org.rubypeople.rdt.refactoring.action.RenameAction,
	handler=ActionDelegateHandlerProxy(null,org.rubypeople.rdt.refactoring.action.RenameAction),
	expression=WorkbenchWindowExpression(org.eclipse.ui.internal.WorkbenchWindow@489479),sourcePriority=16384)
HandlerActivation(commandId=AUTOGEN:::org.rubypeople.rdt.refactoring.refactoringContext/org.rubypeople.rdt.refactoring.action.RenameAction,
	handler=ActionDelegateHandlerProxy(null,org.rubypeople.rdt.refactoring.action.RenameAction),
	expression=WorkbenchWindowExpression(org.eclipse.ui.internal.WorkbenchWindow@489479),sourcePriority=16384)
!SUBENTRY 1 org.eclipse.ui.workbench 2 0 2014-09-06 11:36:06.433
!MESSAGE Conflict for 'AUTOGEN:::org.rubypeople.rdt.refactoring.refactoringContext/org.rubypeople.rdt.refactoring.action.MoveFieldAction':
HandlerActivation(commandId=AUTOGEN:::org.rubypeople.rdt.refactoring.refactoringContext/org.rubypeople.rdt.refactoring.action.MoveFieldAction,
	handler=ActionDelegateHandlerProxy(null,org.rubypeople.rdt.refactoring.action.MoveFieldAction),
	expression=WorkbenchWindowExpression(org.eclipse.ui.internal.WorkbenchWindow@489479),sourcePriority=16384)
HandlerActivation(commandId=AUTOGEN:::org.rubypeople.rdt.refactoring.refactoringContext/org.rubypeople.rdt.refactoring.action.MoveFieldAction,
	handler=ActionDelegateHandlerProxy(null,org.rubypeople.rdt.refactoring.action.MoveFieldAction),
	expression=WorkbenchWindowExpression(org.eclipse.ui.internal.WorkbenchWindow@489479),sourcePriority=16384)
!SUBENTRY 1 org.eclipse.ui.workbench 2 0 2014-09-06 11:36:06.433
!MESSAGE Conflict for 'AUTOGEN:::org.rubypeople.rdt.refactoring.refactoringContext/org.rubypeople.rdt.refactoring.action.EncapsulateFieldAction':
HandlerActivation(commandId=AUTOGEN:::org.rubypeople.rdt.refactoring.refactoringContext/org.rubypeople.rdt.refactoring.action.EncapsulateFieldAction,
	handler=ActionDelegateHandlerProxy(null,org.rubypeople.rdt.refactoring.action.EncapsulateFieldAction),
	expression=WorkbenchWindowExpression(org.eclipse.ui.internal.WorkbenchWindow@489479),sourcePriority=16384)
HandlerActivation(commandId=AUTOGEN:::org.rubypeople.rdt.refactoring.refactoringContext/org.rubypeople.rdt.refactoring.action.EncapsulateFieldAction,
	handler=ActionDelegateHandlerProxy(null,org.rubypeople.rdt.refactoring.action.EncapsulateFieldAction),
	expression=WorkbenchWindowExpression(org.eclipse.ui.internal.WorkbenchWindow@489479),sourcePriority=16384)

!ENTRY org.eclipse.core.resources 4 372 2014-09-06 11:36:07.741
!MESSAGE Resource '/Instituto' is not open.

!ENTRY org.eclipse.core.resources 4 372 2014-09-06 11:36:08.046
!MESSAGE Resource '/Instituto' is not open.

!ENTRY org.eclipse.core.resources 4 372 2014-09-06 11:36:08.151
!MESSAGE Resource '/Instituto' is not open.

!ENTRY org.eclipse.core.resources 4 372 2014-09-06 11:36:08.151
!MESSAGE Resource '/Instituto' is not open.

!ENTRY org.eclipse.core.resources 4 372 2014-09-06 11:36:08.152
!MESSAGE Resource '/Instituto' is not open.

!ENTRY org.eclipse.core.resources 4 372 2014-09-06 11:36:08.153
!MESSAGE Resource '/Instituto' is not open.

!ENTRY org.eclipse.core.resources 4 372 2014-09-06 11:36:08.153
!MESSAGE Resource '/Instituto' is not open.

!ENTRY org.rubypeople.rdt.launching 1 -1 2014-09-06 11:36:08.422
!MESSAGE Was unable to detect default ruby system libraries at: /usr/local

!ENTRY com.aptana.ide.core 4 0 2014-09-06 11:36:08.756
!MESSAGE (Build 2.0.5.1278523018) [ERROR] Unable to connect to database
!STACK 0
SQL Exception: Falha ao criar o banco de dados 'aptanaDB', consulte a próxima exceção para obter detalhes.
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver30.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:233)
	at com.aptana.ide.core.db.AptanaDB.getConnection(AptanaDB.java:225)
	at com.aptana.ide.core.db.FeatureTracker.tableExists(FeatureTracker.java:192)
	at com.aptana.ide.core.db.FeatureTracker.getInstance(FeatureTracker.java:89)
	at com.aptana.ide.core.ui.ping.PingStartup.featuresMatch(PingStartup.java:154)
	at com.aptana.ide.core.ui.ping.PingStartup.addFeatureList(PingStartup.java:111)
	at com.aptana.ide.core.ui.ping.PingStartup.updateAnonymousId(PingStartup.java:391)
	at com.aptana.ide.core.ui.ping.PingStartup.access$0(PingStartup.java:367)
	at com.aptana.ide.core.ui.ping.PingStartup$1.run(PingStartup.java:446)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.aptana.ide.core 4 0 2014-09-06 11:36:08.758
!MESSAGE (Build 2.0.5.1278523018) [ERROR] Unable to connect to database
!STACK 0
SQL Exception: Falha ao iniciar o banco de dados 'aptanaDB', consulte a próxima exceção para obter detalhes.
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver30.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:233)
	at com.aptana.ide.core.db.AptanaDB.getConnection(AptanaDB.java:225)
	at com.aptana.ide.core.db.EventLogger.tableExists(EventLogger.java:290)
	at com.aptana.ide.core.db.EventLogger.getInstance(EventLogger.java:89)
	at org.radrails.rails.ui.RailsUIPlugin$2.run(RailsUIPlugin.java:204)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.aptana.ide.core 4 0 2014-09-06 11:36:08.953
!MESSAGE (Build 2.0.5.1278523018) [ERROR] Unable to connect to database
!STACK 0
SQL Exception: Falha ao iniciar o banco de dados 'aptanaDB', consulte a próxima exceção para obter detalhes.
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver30.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:233)
	at com.aptana.ide.core.db.AptanaDB.getConnection(AptanaDB.java:225)
	at com.aptana.ide.core.db.AptanaDB.execute(AptanaDB.java:97)
	at com.aptana.ide.core.db.FeatureTracker.getInstance(FeatureTracker.java:103)
	at com.aptana.ide.core.ui.ping.PingStartup.featuresMatch(PingStartup.java:154)
	at com.aptana.ide.core.ui.ping.PingStartup.addFeatureList(PingStartup.java:111)
	at com.aptana.ide.core.ui.ping.PingStartup.updateAnonymousId(PingStartup.java:391)
	at com.aptana.ide.core.ui.ping.PingStartup.access$0(PingStartup.java:367)
	at com.aptana.ide.core.ui.ping.PingStartup$1.run(PingStartup.java:446)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.aptana.ide.core 4 0 2014-09-06 11:36:09.591
!MESSAGE (Build 2.0.5.1278523018) [ERROR] Unable to connect to database
!STACK 0
SQL Exception: Falha ao iniciar o banco de dados 'aptanaDB', consulte a próxima exceção para obter detalhes.
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver30.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:233)
	at com.aptana.ide.core.db.AptanaDB.getConnection(AptanaDB.java:225)
	at com.aptana.ide.core.db.AptanaDB.execute(AptanaDB.java:97)
	at com.aptana.ide.core.db.EventLogger.getInstance(EventLogger.java:103)
	at org.radrails.rails.ui.RailsUIPlugin$2.run(RailsUIPlugin.java:204)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.aptana.ide.core 4 0 2014-09-06 11:36:09.734
!MESSAGE (Build 2.0.5.1278523018) [ERROR] Unable to connect to database
!STACK 0
SQL Exception: Falha ao iniciar o banco de dados 'aptanaDB', consulte a próxima exceção para obter detalhes.
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver30.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:233)
	at com.aptana.ide.core.db.AptanaDB.getConnection(AptanaDB.java:225)
	at com.aptana.ide.core.db.AptanaDB.execute(AptanaDB.java:146)
	at com.aptana.ide.core.db.FeatureTracker.getFeatures(FeatureTracker.java:162)
	at com.aptana.ide.core.ui.ping.PingStartup.featuresMatch(PingStartup.java:154)
	at com.aptana.ide.core.ui.ping.PingStartup.addFeatureList(PingStartup.java:111)
	at com.aptana.ide.core.ui.ping.PingStartup.updateAnonymousId(PingStartup.java:391)
	at com.aptana.ide.core.ui.ping.PingStartup.access$0(PingStartup.java:367)
	at com.aptana.ide.core.ui.ping.PingStartup$1.run(PingStartup.java:446)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY org.rubypeople.rdt.launching 4 4 2014-09-06 11:36:09.740
!MESSAGE Failed to retrieve platform for /usr

!ENTRY org.rubypeople.rdt.launching 1 -1 2014-09-06 11:36:09.772
!MESSAGE Starting: /usr/bin/ruby -I /home/joao/Projetos/instituto/.metadata/.plugins/org.rubypeople.rdt.launching/ruby/flush -rrdt_stream_sync.rb -- /usr/bin/gem sources -l

!ENTRY com.aptana.ide.core 4 0 2014-09-06 11:36:09.823
!MESSAGE (Build 2.0.5.1278523018) [ERROR] Unable to connect to database
!STACK 0
SQL Exception: Falha ao iniciar o banco de dados 'aptanaDB', consulte a próxima exceção para obter detalhes.
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver30.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:233)
	at com.aptana.ide.core.db.AptanaDB.getConnection(AptanaDB.java:225)
	at com.aptana.ide.core.db.AptanaDB.execute(AptanaDB.java:97)
	at com.aptana.ide.core.db.FeatureTracker.clearFeatures(FeatureTracker.java:150)
	at com.aptana.ide.core.ui.ping.PingStartup.addFeatureList(PingStartup.java:116)
	at com.aptana.ide.core.ui.ping.PingStartup.updateAnonymousId(PingStartup.java:391)
	at com.aptana.ide.core.ui.ping.PingStartup.access$0(PingStartup.java:367)
	at com.aptana.ide.core.ui.ping.PingStartup$1.run(PingStartup.java:446)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.aptana.ide.core 4 0 2014-09-06 11:36:09.824
!MESSAGE (Build 2.0.5.1278523018) [ERROR] Unable to connect to database
!STACK 0
SQL Exception: Falha ao criar o banco de dados 'aptanaDB', consulte a próxima exceção para obter detalhes.
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver30.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:233)
	at com.aptana.ide.core.db.AptanaDB.getConnection(AptanaDB.java:225)
	at com.aptana.ide.core.db.AptanaDB.execute(AptanaDB.java:97)
	at com.aptana.ide.core.db.EventLogger.logEvent(EventLogger.java:142)
	at org.radrails.rails.ui.RailsUIPlugin$2.run(RailsUIPlugin.java:204)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.aptana.ide.core 4 0 2014-09-06 11:36:09.913
!MESSAGE (Build 2.0.5.1278523018) [ERROR] Unable to connect to database
!STACK 0
SQL Exception: Falha ao iniciar o banco de dados 'aptanaDB', consulte a próxima exceção para obter detalhes.
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver30.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:233)
	at com.aptana.ide.core.db.AptanaDB.getConnection(AptanaDB.java:225)
	at com.aptana.ide.core.db.AptanaDB.execute(AptanaDB.java:97)
	at com.aptana.ide.core.db.EventLogger.logEvent(EventLogger.java:142)
	at com.aptana.ide.core.ui.ping.PingStartup.addFeatureList(PingStartup.java:123)
	at com.aptana.ide.core.ui.ping.PingStartup.updateAnonymousId(PingStartup.java:391)
	at com.aptana.ide.core.ui.ping.PingStartup.access$0(PingStartup.java:367)
	at com.aptana.ide.core.ui.ping.PingStartup$1.run(PingStartup.java:446)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.aptana.ide.core 4 0 2014-09-06 11:36:10.037
!MESSAGE (Build 2.0.5.1278523018) [ERROR] Unable to connect to database
!STACK 0
SQL Exception: Falha ao iniciar o banco de dados 'aptanaDB', consulte a próxima exceção para obter detalhes.
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver30.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:233)
	at com.aptana.ide.core.db.AptanaDB.getConnection(AptanaDB.java:225)
	at com.aptana.ide.core.db.AptanaDB.execute(AptanaDB.java:97)
	at com.aptana.ide.core.db.FeatureTracker.addFeature(FeatureTracker.java:140)
	at com.aptana.ide.core.db.FeatureTracker.addFeature(FeatureTracker.java:117)
	at com.aptana.ide.core.ui.ping.PingStartup.addFeatureList(PingStartup.java:124)
	at com.aptana.ide.core.ui.ping.PingStartup.updateAnonymousId(PingStartup.java:391)
	at com.aptana.ide.core.ui.ping.PingStartup.access$0(PingStartup.java:367)
	at com.aptana.ide.core.ui.ping.PingStartup$1.run(PingStartup.java:446)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY org.rubypeople.rdt.launching 4 4 2014-09-06 11:36:10.093
!MESSAGE Failed to retrieve platform for /usr

!ENTRY org.rubypeople.rdt.launching 1 -1 2014-09-06 11:36:10.106
!MESSAGE Starting: /usr/bin/ruby -I /home/joao/Projetos/instituto/.metadata/.plugins/org.rubypeople.rdt.launching/ruby/flush -rrdt_stream_sync.rb -- /usr/bin/gem -v

!ENTRY com.aptana.ide.core 4 0 2014-09-06 11:36:10.148
!MESSAGE (Build 2.0.5.1278523018) [ERROR] Unable to connect to database
!STACK 0
SQL Exception: Falha ao iniciar o banco de dados 'aptanaDB', consulte a próxima exceção para obter detalhes.
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver30.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:233)
	at com.aptana.ide.core.db.AptanaDB.getConnection(AptanaDB.java:225)
	at com.aptana.ide.core.db.AptanaDB.execute(AptanaDB.java:97)
	at com.aptana.ide.core.db.EventLogger.logEvent(EventLogger.java:142)
	at com.aptana.ide.core.ui.ping.PingStartup.addFeatureList(PingStartup.java:123)
	at com.aptana.ide.core.ui.ping.PingStartup.updateAnonymousId(PingStartup.java:391)
	at com.aptana.ide.core.ui.ping.PingStartup.access$0(PingStartup.java:367)
	at com.aptana.ide.core.ui.ping.PingStartup$1.run(PingStartup.java:446)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.aptana.ide.core 4 0 2014-09-06 11:36:10.249
!MESSAGE (Build 2.0.5.1278523018) [ERROR] Unable to connect to database
!STACK 0
SQL Exception: Falha ao iniciar o banco de dados 'aptanaDB', consulte a próxima exceção para obter detalhes.
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver30.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:233)
	at com.aptana.ide.core.db.AptanaDB.getConnection(AptanaDB.java:225)
	at com.aptana.ide.core.db.AptanaDB.execute(AptanaDB.java:97)
	at com.aptana.ide.core.db.FeatureTracker.addFeature(FeatureTracker.java:140)
	at com.aptana.ide.core.db.FeatureTracker.addFeature(FeatureTracker.java:117)
	at com.aptana.ide.core.ui.ping.PingStartup.addFeatureList(PingStartup.java:124)
	at com.aptana.ide.core.ui.ping.PingStartup.updateAnonymousId(PingStartup.java:391)
	at com.aptana.ide.core.ui.ping.PingStartup.access$0(PingStartup.java:367)
	at com.aptana.ide.core.ui.ping.PingStartup$1.run(PingStartup.java:446)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.aptana.ide.core 4 0 2014-09-06 11:36:10.338
!MESSAGE (Build 2.0.5.1278523018) [ERROR] Unable to connect to database
!STACK 0
SQL Exception: Falha ao iniciar o banco de dados 'aptanaDB', consulte a próxima exceção para obter detalhes.
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver30.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:233)
	at com.aptana.ide.core.db.AptanaDB.getConnection(AptanaDB.java:225)
	at com.aptana.ide.core.db.AptanaDB.execute(AptanaDB.java:97)
	at com.aptana.ide.core.db.EventLogger.logEvent(EventLogger.java:142)
	at com.aptana.ide.core.ui.ping.PingStartup.addFeatureList(PingStartup.java:123)
	at com.aptana.ide.core.ui.ping.PingStartup.updateAnonymousId(PingStartup.java:391)
	at com.aptana.ide.core.ui.ping.PingStartup.access$0(PingStartup.java:367)
	at com.aptana.ide.core.ui.ping.PingStartup$1.run(PingStartup.java:446)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.aptana.ide.core 4 0 2014-09-06 11:36:10.417
!MESSAGE (Build 2.0.5.1278523018) [ERROR] Unable to connect to database
!STACK 0
SQL Exception: Falha ao iniciar o banco de dados 'aptanaDB', consulte a próxima exceção para obter detalhes.
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver30.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:233)
	at com.aptana.ide.core.db.AptanaDB.getConnection(AptanaDB.java:225)
	at com.aptana.ide.core.db.AptanaDB.execute(AptanaDB.java:97)
	at com.aptana.ide.core.db.FeatureTracker.addFeature(FeatureTracker.java:140)
	at com.aptana.ide.core.db.FeatureTracker.addFeature(FeatureTracker.java:117)
	at com.aptana.ide.core.ui.ping.PingStartup.addFeatureList(PingStartup.java:124)
	at com.aptana.ide.core.ui.ping.PingStartup.updateAnonymousId(PingStartup.java:391)
	at com.aptana.ide.core.ui.ping.PingStartup.access$0(PingStartup.java:367)
	at com.aptana.ide.core.ui.ping.PingStartup$1.run(PingStartup.java:446)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY org.rubypeople.rdt.launching 4 4 2014-09-06 11:36:10.453
!MESSAGE Failed to retrieve platform for /usr

!ENTRY org.rubypeople.rdt.launching 1 -1 2014-09-06 11:36:10.466
!MESSAGE Starting: /usr/bin/ruby -I /home/joao/Projetos/instituto/.metadata/.plugins/org.rubypeople.rdt.launching/ruby/flush -rrdt_stream_sync.rb -- /usr/bin/gem query -d

!ENTRY com.aptana.ide.core 4 0 2014-09-06 11:36:10.507
!MESSAGE (Build 2.0.5.1278523018) [ERROR] Unable to connect to database
!STACK 0
SQL Exception: Falha ao iniciar o banco de dados 'aptanaDB', consulte a próxima exceção para obter detalhes.
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver30.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:233)
	at com.aptana.ide.core.db.AptanaDB.getConnection(AptanaDB.java:225)
	at com.aptana.ide.core.db.AptanaDB.execute(AptanaDB.java:97)
	at com.aptana.ide.core.db.EventLogger.logEvent(EventLogger.java:142)
	at com.aptana.ide.core.ui.ping.PingStartup.addFeatureList(PingStartup.java:123)
	at com.aptana.ide.core.ui.ping.PingStartup.updateAnonymousId(PingStartup.java:391)
	at com.aptana.ide.core.ui.ping.PingStartup.access$0(PingStartup.java:367)
	at com.aptana.ide.core.ui.ping.PingStartup$1.run(PingStartup.java:446)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.aptana.ide.core 4 0 2014-09-06 11:36:10.596
!MESSAGE (Build 2.0.5.1278523018) [ERROR] Unable to connect to database
!STACK 0
SQL Exception: Falha ao iniciar o banco de dados 'aptanaDB', consulte a próxima exceção para obter detalhes.
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver30.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:233)
	at com.aptana.ide.core.db.AptanaDB.getConnection(AptanaDB.java:225)
	at com.aptana.ide.core.db.AptanaDB.execute(AptanaDB.java:97)
	at com.aptana.ide.core.db.FeatureTracker.addFeature(FeatureTracker.java:140)
	at com.aptana.ide.core.db.FeatureTracker.addFeature(FeatureTracker.java:117)
	at com.aptana.ide.core.ui.ping.PingStartup.addFeatureList(PingStartup.java:124)
	at com.aptana.ide.core.ui.ping.PingStartup.updateAnonymousId(PingStartup.java:391)
	at com.aptana.ide.core.ui.ping.PingStartup.access$0(PingStartup.java:367)
	at com.aptana.ide.core.ui.ping.PingStartup$1.run(PingStartup.java:446)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.aptana.ide.core 4 0 2014-09-06 11:36:10.687
!MESSAGE (Build 2.0.5.1278523018) [ERROR] Unable to connect to database
!STACK 0
SQL Exception: Falha ao iniciar o banco de dados 'aptanaDB', consulte a próxima exceção para obter detalhes.
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver30.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:233)
	at com.aptana.ide.core.db.AptanaDB.getConnection(AptanaDB.java:225)
	at com.aptana.ide.core.db.AptanaDB.execute(AptanaDB.java:97)
	at com.aptana.ide.core.db.EventLogger.logEvent(EventLogger.java:142)
	at com.aptana.ide.core.ui.ping.PingStartup.addFeatureList(PingStartup.java:123)
	at com.aptana.ide.core.ui.ping.PingStartup.updateAnonymousId(PingStartup.java:391)
	at com.aptana.ide.core.ui.ping.PingStartup.access$0(PingStartup.java:367)
	at com.aptana.ide.core.ui.ping.PingStartup$1.run(PingStartup.java:446)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY org.rubypeople.rdt.launching 4 4 2014-09-06 11:36:10.775
!MESSAGE Failed to retrieve platform for /usr

!ENTRY com.aptana.ide.core 4 0 2014-09-06 11:36:10.788
!MESSAGE (Build 2.0.5.1278523018) [ERROR] Unable to connect to database
!STACK 0
SQL Exception: Falha ao iniciar o banco de dados 'aptanaDB', consulte a próxima exceção para obter detalhes.
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver30.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:233)
	at com.aptana.ide.core.db.AptanaDB.getConnection(AptanaDB.java:225)
	at com.aptana.ide.core.db.AptanaDB.execute(AptanaDB.java:97)
	at com.aptana.ide.core.db.FeatureTracker.addFeature(FeatureTracker.java:140)
	at com.aptana.ide.core.db.FeatureTracker.addFeature(FeatureTracker.java:117)
	at com.aptana.ide.core.ui.ping.PingStartup.addFeatureList(PingStartup.java:124)
	at com.aptana.ide.core.ui.ping.PingStartup.updateAnonymousId(PingStartup.java:391)
	at com.aptana.ide.core.ui.ping.PingStartup.access$0(PingStartup.java:367)
	at com.aptana.ide.core.ui.ping.PingStartup$1.run(PingStartup.java:446)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY org.rubypeople.rdt.launching 4 4 2014-09-06 11:36:10.836
!MESSAGE Failed to retrieve platform for /usr

!ENTRY com.aptana.ide.core 4 0 2014-09-06 11:36:10.877
!MESSAGE (Build 2.0.5.1278523018) [ERROR] Unable to connect to database
!STACK 0
SQL Exception: Falha ao iniciar o banco de dados 'aptanaDB', consulte a próxima exceção para obter detalhes.
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver30.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:233)
	at com.aptana.ide.core.db.AptanaDB.getConnection(AptanaDB.java:225)
	at com.aptana.ide.core.db.AptanaDB.execute(AptanaDB.java:97)
	at com.aptana.ide.core.db.EventLogger.logEvent(EventLogger.java:142)
	at com.aptana.ide.core.ui.ping.PingStartup.addFeatureList(PingStartup.java:123)
	at com.aptana.ide.core.ui.ping.PingStartup.updateAnonymousId(PingStartup.java:391)
	at com.aptana.ide.core.ui.ping.PingStartup.access$0(PingStartup.java:367)
	at com.aptana.ide.core.ui.ping.PingStartup$1.run(PingStartup.java:446)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY org.rubypeople.rdt.launching 4 4 2014-09-06 11:36:10.895
!MESSAGE Failed to retrieve platform for /usr

!ENTRY org.rubypeople.rdt.launching 4 4 2014-09-06 11:36:10.961
!MESSAGE Failed to retrieve platform for /usr

!ENTRY com.aptana.ide.core 4 0 2014-09-06 11:36:10.967
!MESSAGE (Build 2.0.5.1278523018) [ERROR] Unable to connect to database
!STACK 0
SQL Exception: Falha ao iniciar o banco de dados 'aptanaDB', consulte a próxima exceção para obter detalhes.
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver30.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:233)
	at com.aptana.ide.core.db.AptanaDB.getConnection(AptanaDB.java:225)
	at com.aptana.ide.core.db.AptanaDB.execute(AptanaDB.java:97)
	at com.aptana.ide.core.db.FeatureTracker.addFeature(FeatureTracker.java:140)
	at com.aptana.ide.core.db.FeatureTracker.addFeature(FeatureTracker.java:117)
	at com.aptana.ide.core.ui.ping.PingStartup.addFeatureList(PingStartup.java:124)
	at com.aptana.ide.core.ui.ping.PingStartup.updateAnonymousId(PingStartup.java:391)
	at com.aptana.ide.core.ui.ping.PingStartup.access$0(PingStartup.java:367)
	at com.aptana.ide.core.ui.ping.PingStartup$1.run(PingStartup.java:446)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.aptana.ide.core 4 0 2014-09-06 11:36:11.135
!MESSAGE (Build 2.0.5.1278523018) [ERROR] Unable to connect to database
!STACK 0
SQL Exception: Falha ao iniciar o banco de dados 'aptanaDB', consulte a próxima exceção para obter detalhes.
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver30.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:233)
	at com.aptana.ide.core.db.AptanaDB.getConnection(AptanaDB.java:225)
	at com.aptana.ide.core.db.AptanaDB.execute(AptanaDB.java:97)
	at com.aptana.ide.core.db.EventLogger.logEvent(EventLogger.java:142)
	at com.aptana.ide.core.ui.ping.PingStartup.addFeatureList(PingStartup.java:123)
	at com.aptana.ide.core.ui.ping.PingStartup.updateAnonymousId(PingStartup.java:391)
	at com.aptana.ide.core.ui.ping.PingStartup.access$0(PingStartup.java:367)
	at com.aptana.ide.core.ui.ping.PingStartup$1.run(PingStartup.java:446)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.aptana.ide.core 4 0 2014-09-06 11:36:11.223
!MESSAGE (Build 2.0.5.1278523018) [ERROR] Unable to connect to database
!STACK 0
SQL Exception: Falha ao iniciar o banco de dados 'aptanaDB', consulte a próxima exceção para obter detalhes.
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver30.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:233)
	at com.aptana.ide.core.db.AptanaDB.getConnection(AptanaDB.java:225)
	at com.aptana.ide.core.db.AptanaDB.execute(AptanaDB.java:97)
	at com.aptana.ide.core.db.FeatureTracker.addFeature(FeatureTracker.java:140)
	at com.aptana.ide.core.db.FeatureTracker.addFeature(FeatureTracker.java:117)
	at com.aptana.ide.core.ui.ping.PingStartup.addFeatureList(PingStartup.java:124)
	at com.aptana.ide.core.ui.ping.PingStartup.updateAnonymousId(PingStartup.java:391)
	at com.aptana.ide.core.ui.ping.PingStartup.access$0(PingStartup.java:367)
	at com.aptana.ide.core.ui.ping.PingStartup$1.run(PingStartup.java:446)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.aptana.ide.core 4 0 2014-09-06 11:36:11.323
!MESSAGE (Build 2.0.5.1278523018) [ERROR] Unable to connect to database
!STACK 0
SQL Exception: Falha ao iniciar o banco de dados 'aptanaDB', consulte a próxima exceção para obter detalhes.
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver30.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:233)
	at com.aptana.ide.core.db.AptanaDB.getConnection(AptanaDB.java:225)
	at com.aptana.ide.core.db.AptanaDB.execute(AptanaDB.java:97)
	at com.aptana.ide.core.db.EventLogger.logEvent(EventLogger.java:142)
	at com.aptana.ide.core.ui.ping.PingStartup.addFeatureList(PingStartup.java:123)
	at com.aptana.ide.core.ui.ping.PingStartup.updateAnonymousId(PingStartup.java:391)
	at com.aptana.ide.core.ui.ping.PingStartup.access$0(PingStartup.java:367)
	at com.aptana.ide.core.ui.ping.PingStartup$1.run(PingStartup.java:446)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.aptana.ide.core 4 0 2014-09-06 11:36:11.412
!MESSAGE (Build 2.0.5.1278523018) [ERROR] Unable to connect to database
!STACK 0
SQL Exception: Falha ao iniciar o banco de dados 'aptanaDB', consulte a próxima exceção para obter detalhes.
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver30.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:233)
	at com.aptana.ide.core.db.AptanaDB.getConnection(AptanaDB.java:225)
	at com.aptana.ide.core.db.AptanaDB.execute(AptanaDB.java:97)
	at com.aptana.ide.core.db.FeatureTracker.addFeature(FeatureTracker.java:140)
	at com.aptana.ide.core.db.FeatureTracker.addFeature(FeatureTracker.java:117)
	at com.aptana.ide.core.ui.ping.PingStartup.addFeatureList(PingStartup.java:124)
	at com.aptana.ide.core.ui.ping.PingStartup.updateAnonymousId(PingStartup.java:391)
	at com.aptana.ide.core.ui.ping.PingStartup.access$0(PingStartup.java:367)
	at com.aptana.ide.core.ui.ping.PingStartup$1.run(PingStartup.java:446)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.aptana.ide.core 4 0 2014-09-06 11:36:11.515
!MESSAGE (Build 2.0.5.1278523018) [ERROR] Unable to connect to database
!STACK 0
SQL Exception: Falha ao iniciar o banco de dados 'aptanaDB', consulte a próxima exceção para obter detalhes.
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver30.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:233)
	at com.aptana.ide.core.db.AptanaDB.getConnection(AptanaDB.java:225)
	at com.aptana.ide.core.db.AptanaDB.execute(AptanaDB.java:97)
	at com.aptana.ide.core.db.EventLogger.logEvent(EventLogger.java:142)
	at com.aptana.ide.core.ui.ping.PingStartup.addFeatureList(PingStartup.java:123)
	at com.aptana.ide.core.ui.ping.PingStartup.updateAnonymousId(PingStartup.java:391)
	at com.aptana.ide.core.ui.ping.PingStartup.access$0(PingStartup.java:367)
	at com.aptana.ide.core.ui.ping.PingStartup$1.run(PingStartup.java:446)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.aptana.ide.core 4 0 2014-09-06 11:36:11.593
!MESSAGE (Build 2.0.5.1278523018) [ERROR] Unable to connect to database
!STACK 0
SQL Exception: Falha ao iniciar o banco de dados 'aptanaDB', consulte a próxima exceção para obter detalhes.
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver30.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:233)
	at com.aptana.ide.core.db.AptanaDB.getConnection(AptanaDB.java:225)
	at com.aptana.ide.core.db.AptanaDB.execute(AptanaDB.java:97)
	at com.aptana.ide.core.db.FeatureTracker.addFeature(FeatureTracker.java:140)
	at com.aptana.ide.core.db.FeatureTracker.addFeature(FeatureTracker.java:117)
	at com.aptana.ide.core.ui.ping.PingStartup.addFeatureList(PingStartup.java:124)
	at com.aptana.ide.core.ui.ping.PingStartup.updateAnonymousId(PingStartup.java:391)
	at com.aptana.ide.core.ui.ping.PingStartup.access$0(PingStartup.java:367)
	at com.aptana.ide.core.ui.ping.PingStartup$1.run(PingStartup.java:446)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.aptana.ide.core 4 0 2014-09-06 11:36:11.682
!MESSAGE (Build 2.0.5.1278523018) [ERROR] Unable to connect to database
!STACK 0
SQL Exception: Falha ao iniciar o banco de dados 'aptanaDB', consulte a próxima exceção para obter detalhes.
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver30.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:233)
	at com.aptana.ide.core.db.AptanaDB.getConnection(AptanaDB.java:225)
	at com.aptana.ide.core.db.AptanaDB.execute(AptanaDB.java:97)
	at com.aptana.ide.core.db.EventLogger.logEvent(EventLogger.java:142)
	at com.aptana.ide.core.ui.ping.PingStartup.addFeatureList(PingStartup.java:123)
	at com.aptana.ide.core.ui.ping.PingStartup.updateAnonymousId(PingStartup.java:391)
	at com.aptana.ide.core.ui.ping.PingStartup.access$0(PingStartup.java:367)
	at com.aptana.ide.core.ui.ping.PingStartup$1.run(PingStartup.java:446)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.aptana.ide.core 4 0 2014-09-06 11:36:11.771
!MESSAGE (Build 2.0.5.1278523018) [ERROR] Unable to connect to database
!STACK 0
SQL Exception: Falha ao iniciar o banco de dados 'aptanaDB', consulte a próxima exceção para obter detalhes.
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver30.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:233)
	at com.aptana.ide.core.db.AptanaDB.getConnection(AptanaDB.java:225)
	at com.aptana.ide.core.db.AptanaDB.execute(AptanaDB.java:97)
	at com.aptana.ide.core.db.FeatureTracker.addFeature(FeatureTracker.java:140)
	at com.aptana.ide.core.db.FeatureTracker.addFeature(FeatureTracker.java:117)
	at com.aptana.ide.core.ui.ping.PingStartup.addFeatureList(PingStartup.java:124)
	at com.aptana.ide.core.ui.ping.PingStartup.updateAnonymousId(PingStartup.java:391)
	at com.aptana.ide.core.ui.ping.PingStartup.access$0(PingStartup.java:367)
	at com.aptana.ide.core.ui.ping.PingStartup$1.run(PingStartup.java:446)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.aptana.ide.core 4 0 2014-09-06 11:36:11.861
!MESSAGE (Build 2.0.5.1278523018) [ERROR] Unable to connect to database
!STACK 0
SQL Exception: Falha ao iniciar o banco de dados 'aptanaDB', consulte a próxima exceção para obter detalhes.
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver30.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:233)
	at com.aptana.ide.core.db.AptanaDB.getConnection(AptanaDB.java:225)
	at com.aptana.ide.core.db.AptanaDB.execute(AptanaDB.java:97)
	at com.aptana.ide.core.db.EventLogger.logEvent(EventLogger.java:142)
	at com.aptana.ide.core.ui.ping.PingStartup.addFeatureList(PingStartup.java:123)
	at com.aptana.ide.core.ui.ping.PingStartup.updateAnonymousId(PingStartup.java:391)
	at com.aptana.ide.core.ui.ping.PingStartup.access$0(PingStartup.java:367)
	at com.aptana.ide.core.ui.ping.PingStartup$1.run(PingStartup.java:446)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.aptana.ide.core 4 0 2014-09-06 11:36:11.950
!MESSAGE (Build 2.0.5.1278523018) [ERROR] Unable to connect to database
!STACK 0
SQL Exception: Falha ao iniciar o banco de dados 'aptanaDB', consulte a próxima exceção para obter detalhes.
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver30.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:233)
	at com.aptana.ide.core.db.AptanaDB.getConnection(AptanaDB.java:225)
	at com.aptana.ide.core.db.AptanaDB.execute(AptanaDB.java:97)
	at com.aptana.ide.core.db.FeatureTracker.addFeature(FeatureTracker.java:140)
	at com.aptana.ide.core.db.FeatureTracker.addFeature(FeatureTracker.java:117)
	at com.aptana.ide.core.ui.ping.PingStartup.addFeatureList(PingStartup.java:124)
	at com.aptana.ide.core.ui.ping.PingStartup.updateAnonymousId(PingStartup.java:391)
	at com.aptana.ide.core.ui.ping.PingStartup.access$0(PingStartup.java:367)
	at com.aptana.ide.core.ui.ping.PingStartup$1.run(PingStartup.java:446)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.aptana.ide.core 4 0 2014-09-06 11:36:12.040
!MESSAGE (Build 2.0.5.1278523018) [ERROR] Unable to connect to database
!STACK 0
SQL Exception: Falha ao iniciar o banco de dados 'aptanaDB', consulte a próxima exceção para obter detalhes.
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver30.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:233)
	at com.aptana.ide.core.db.AptanaDB.getConnection(AptanaDB.java:225)
	at com.aptana.ide.core.db.AptanaDB.execute(AptanaDB.java:97)
	at com.aptana.ide.core.db.EventLogger.logEvent(EventLogger.java:142)
	at com.aptana.ide.core.ui.ping.PingStartup.addFeatureList(PingStartup.java:123)
	at com.aptana.ide.core.ui.ping.PingStartup.updateAnonymousId(PingStartup.java:391)
	at com.aptana.ide.core.ui.ping.PingStartup.access$0(PingStartup.java:367)
	at com.aptana.ide.core.ui.ping.PingStartup$1.run(PingStartup.java:446)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.aptana.ide.core 4 0 2014-09-06 11:36:12.129
!MESSAGE (Build 2.0.5.1278523018) [ERROR] Unable to connect to database
!STACK 0
SQL Exception: Falha ao iniciar o banco de dados 'aptanaDB', consulte a próxima exceção para obter detalhes.
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver30.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:233)
	at com.aptana.ide.core.db.AptanaDB.getConnection(AptanaDB.java:225)
	at com.aptana.ide.core.db.AptanaDB.execute(AptanaDB.java:97)
	at com.aptana.ide.core.db.FeatureTracker.addFeature(FeatureTracker.java:140)
	at com.aptana.ide.core.db.FeatureTracker.addFeature(FeatureTracker.java:117)
	at com.aptana.ide.core.ui.ping.PingStartup.addFeatureList(PingStartup.java:124)
	at com.aptana.ide.core.ui.ping.PingStartup.updateAnonymousId(PingStartup.java:391)
	at com.aptana.ide.core.ui.ping.PingStartup.access$0(PingStartup.java:367)
	at com.aptana.ide.core.ui.ping.PingStartup$1.run(PingStartup.java:446)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.aptana.ide.core 4 0 2014-09-06 11:36:12.242
!MESSAGE (Build 2.0.5.1278523018) [ERROR] Unable to connect to database
!STACK 0
SQL Exception: Falha ao iniciar o banco de dados 'aptanaDB', consulte a próxima exceção para obter detalhes.
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver30.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:233)
	at com.aptana.ide.core.db.AptanaDB.getConnection(AptanaDB.java:225)
	at com.aptana.ide.core.db.AptanaDB.execute(AptanaDB.java:97)
	at com.aptana.ide.core.db.EventLogger.logEvent(EventLogger.java:142)
	at com.aptana.ide.core.ui.ping.PingStartup.addFeatureList(PingStartup.java:123)
	at com.aptana.ide.core.ui.ping.PingStartup.updateAnonymousId(PingStartup.java:391)
	at com.aptana.ide.core.ui.ping.PingStartup.access$0(PingStartup.java:367)
	at com.aptana.ide.core.ui.ping.PingStartup$1.run(PingStartup.java:446)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.aptana.ide.core 4 0 2014-09-06 11:36:12.320
!MESSAGE (Build 2.0.5.1278523018) [ERROR] Unable to connect to database
!STACK 0
SQL Exception: Falha ao iniciar o banco de dados 'aptanaDB', consulte a próxima exceção para obter detalhes.
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver30.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:233)
	at com.aptana.ide.core.db.AptanaDB.getConnection(AptanaDB.java:225)
	at com.aptana.ide.core.db.AptanaDB.execute(AptanaDB.java:97)
	at com.aptana.ide.core.db.FeatureTracker.addFeature(FeatureTracker.java:140)
	at com.aptana.ide.core.db.FeatureTracker.addFeature(FeatureTracker.java:117)
	at com.aptana.ide.core.ui.ping.PingStartup.addFeatureList(PingStartup.java:124)
	at com.aptana.ide.core.ui.ping.PingStartup.updateAnonymousId(PingStartup.java:391)
	at com.aptana.ide.core.ui.ping.PingStartup.access$0(PingStartup.java:367)
	at com.aptana.ide.core.ui.ping.PingStartup$1.run(PingStartup.java:446)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.aptana.ide.core 4 0 2014-09-06 11:36:12.388
!MESSAGE (Build 2.0.5.1278523018) [ERROR] Unable to connect to database
!STACK 0
SQL Exception: Falha ao iniciar o banco de dados 'aptanaDB', consulte a próxima exceção para obter detalhes.
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver30.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:233)
	at com.aptana.ide.core.db.AptanaDB.getConnection(AptanaDB.java:225)
	at com.aptana.ide.core.db.AptanaDB.execute(AptanaDB.java:97)
	at com.aptana.ide.core.db.EventLogger.logEvent(EventLogger.java:142)
	at com.aptana.ide.core.ui.ping.PingStartup.addFeatureList(PingStartup.java:123)
	at com.aptana.ide.core.ui.ping.PingStartup.updateAnonymousId(PingStartup.java:391)
	at com.aptana.ide.core.ui.ping.PingStartup.access$0(PingStartup.java:367)
	at com.aptana.ide.core.ui.ping.PingStartup$1.run(PingStartup.java:446)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.aptana.ide.core 4 0 2014-09-06 11:36:12.477
!MESSAGE (Build 2.0.5.1278523018) [ERROR] Unable to connect to database
!STACK 0
SQL Exception: Falha ao iniciar o banco de dados 'aptanaDB', consulte a próxima exceção para obter detalhes.
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver30.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:233)
	at com.aptana.ide.core.db.AptanaDB.getConnection(AptanaDB.java:225)
	at com.aptana.ide.core.db.AptanaDB.execute(AptanaDB.java:97)
	at com.aptana.ide.core.db.FeatureTracker.addFeature(FeatureTracker.java:140)
	at com.aptana.ide.core.db.FeatureTracker.addFeature(FeatureTracker.java:117)
	at com.aptana.ide.core.ui.ping.PingStartup.addFeatureList(PingStartup.java:124)
	at com.aptana.ide.core.ui.ping.PingStartup.updateAnonymousId(PingStartup.java:391)
	at com.aptana.ide.core.ui.ping.PingStartup.access$0(PingStartup.java:367)
	at com.aptana.ide.core.ui.ping.PingStartup$1.run(PingStartup.java:446)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.aptana.ide.core 4 0 2014-09-06 11:36:12.566
!MESSAGE (Build 2.0.5.1278523018) [ERROR] Unable to connect to database
!STACK 0
SQL Exception: Falha ao iniciar o banco de dados 'aptanaDB', consulte a próxima exceção para obter detalhes.
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver30.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:233)
	at com.aptana.ide.core.db.AptanaDB.getConnection(AptanaDB.java:225)
	at com.aptana.ide.core.db.AptanaDB.execute(AptanaDB.java:97)
	at com.aptana.ide.core.db.EventLogger.logEvent(EventLogger.java:142)
	at com.aptana.ide.core.ui.ping.PingStartup.addFeatureList(PingStartup.java:123)
	at com.aptana.ide.core.ui.ping.PingStartup.updateAnonymousId(PingStartup.java:391)
	at com.aptana.ide.core.ui.ping.PingStartup.access$0(PingStartup.java:367)
	at com.aptana.ide.core.ui.ping.PingStartup$1.run(PingStartup.java:446)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.aptana.ide.core 4 0 2014-09-06 11:36:12.656
!MESSAGE (Build 2.0.5.1278523018) [ERROR] Unable to connect to database
!STACK 0
SQL Exception: Falha ao iniciar o banco de dados 'aptanaDB', consulte a próxima exceção para obter detalhes.
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver30.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:233)
	at com.aptana.ide.core.db.AptanaDB.getConnection(AptanaDB.java:225)
	at com.aptana.ide.core.db.AptanaDB.execute(AptanaDB.java:97)
	at com.aptana.ide.core.db.FeatureTracker.addFeature(FeatureTracker.java:140)
	at com.aptana.ide.core.db.FeatureTracker.addFeature(FeatureTracker.java:117)
	at com.aptana.ide.core.ui.ping.PingStartup.addFeatureList(PingStartup.java:124)
	at com.aptana.ide.core.ui.ping.PingStartup.updateAnonymousId(PingStartup.java:391)
	at com.aptana.ide.core.ui.ping.PingStartup.access$0(PingStartup.java:367)
	at com.aptana.ide.core.ui.ping.PingStartup$1.run(PingStartup.java:446)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.aptana.ide.core 4 0 2014-09-06 11:36:12.745
!MESSAGE (Build 2.0.5.1278523018) [ERROR] Unable to connect to database
!STACK 0
SQL Exception: Falha ao iniciar o banco de dados 'aptanaDB', consulte a próxima exceção para obter detalhes.
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver30.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:233)
	at com.aptana.ide.core.db.AptanaDB.getConnection(AptanaDB.java:225)
	at com.aptana.ide.core.db.AptanaDB.execute(AptanaDB.java:97)
	at com.aptana.ide.core.db.EventLogger.logEvent(EventLogger.java:142)
	at com.aptana.ide.core.ui.ping.PingStartup.addFeatureList(PingStartup.java:123)
	at com.aptana.ide.core.ui.ping.PingStartup.updateAnonymousId(PingStartup.java:391)
	at com.aptana.ide.core.ui.ping.PingStartup.access$0(PingStartup.java:367)
	at com.aptana.ide.core.ui.ping.PingStartup$1.run(PingStartup.java:446)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.aptana.ide.core 4 0 2014-09-06 11:36:12.835
!MESSAGE (Build 2.0.5.1278523018) [ERROR] Unable to connect to database
!STACK 0
SQL Exception: Falha ao iniciar o banco de dados 'aptanaDB', consulte a próxima exceção para obter detalhes.
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver30.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:233)
	at com.aptana.ide.core.db.AptanaDB.getConnection(AptanaDB.java:225)
	at com.aptana.ide.core.db.AptanaDB.execute(AptanaDB.java:97)
	at com.aptana.ide.core.db.FeatureTracker.addFeature(FeatureTracker.java:140)
	at com.aptana.ide.core.db.FeatureTracker.addFeature(FeatureTracker.java:117)
	at com.aptana.ide.core.ui.ping.PingStartup.addFeatureList(PingStartup.java:124)
	at com.aptana.ide.core.ui.ping.PingStartup.updateAnonymousId(PingStartup.java:391)
	at com.aptana.ide.core.ui.ping.PingStartup.access$0(PingStartup.java:367)
	at com.aptana.ide.core.ui.ping.PingStartup$1.run(PingStartup.java:446)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.aptana.ide.core 4 0 2014-09-06 11:36:12.926
!MESSAGE (Build 2.0.5.1278523018) [ERROR] Unable to connect to database
!STACK 0
SQL Exception: Falha ao iniciar o banco de dados 'aptanaDB', consulte a próxima exceção para obter detalhes.
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver30.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:233)
	at com.aptana.ide.core.db.AptanaDB.getConnection(AptanaDB.java:225)
	at com.aptana.ide.core.db.AptanaDB.execute(AptanaDB.java:97)
	at com.aptana.ide.core.db.EventLogger.logEvent(EventLogger.java:142)
	at com.aptana.ide.core.ui.ping.PingStartup.addFeatureList(PingStartup.java:123)
	at com.aptana.ide.core.ui.ping.PingStartup.updateAnonymousId(PingStartup.java:391)
	at com.aptana.ide.core.ui.ping.PingStartup.access$0(PingStartup.java:367)
	at com.aptana.ide.core.ui.ping.PingStartup$1.run(PingStartup.java:446)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.aptana.ide.core 4 0 2014-09-06 11:36:13.004
!MESSAGE (Build 2.0.5.1278523018) [ERROR] Unable to connect to database
!STACK 0
SQL Exception: Falha ao iniciar o banco de dados 'aptanaDB', consulte a próxima exceção para obter detalhes.
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver30.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:233)
	at com.aptana.ide.core.db.AptanaDB.getConnection(AptanaDB.java:225)
	at com.aptana.ide.core.db.AptanaDB.execute(AptanaDB.java:97)
	at com.aptana.ide.core.db.FeatureTracker.addFeature(FeatureTracker.java:140)
	at com.aptana.ide.core.db.FeatureTracker.addFeature(FeatureTracker.java:117)
	at com.aptana.ide.core.ui.ping.PingStartup.addFeatureList(PingStartup.java:124)
	at com.aptana.ide.core.ui.ping.PingStartup.updateAnonymousId(PingStartup.java:391)
	at com.aptana.ide.core.ui.ping.PingStartup.access$0(PingStartup.java:367)
	at com.aptana.ide.core.ui.ping.PingStartup$1.run(PingStartup.java:446)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.aptana.ide.core 4 0 2014-09-06 11:36:13.155
!MESSAGE (Build 2.0.5.1278523018) [ERROR] Unable to connect to database
!STACK 0
SQL Exception: Falha ao iniciar o banco de dados 'aptanaDB', consulte a próxima exceção para obter detalhes.
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver30.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:233)
	at com.aptana.ide.core.db.AptanaDB.getConnection(AptanaDB.java:225)
	at com.aptana.ide.core.db.AptanaDB.execute(AptanaDB.java:97)
	at com.aptana.ide.core.db.EventLogger.logEvent(EventLogger.java:142)
	at com.aptana.ide.core.ui.ping.PingStartup.addFeatureList(PingStartup.java:123)
	at com.aptana.ide.core.ui.ping.PingStartup.updateAnonymousId(PingStartup.java:391)
	at com.aptana.ide.core.ui.ping.PingStartup.access$0(PingStartup.java:367)
	at com.aptana.ide.core.ui.ping.PingStartup$1.run(PingStartup.java:446)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.aptana.ide.core 4 0 2014-09-06 11:36:13.238
!MESSAGE (Build 2.0.5.1278523018) [ERROR] Unable to connect to database
!STACK 0
SQL Exception: Falha ao iniciar o banco de dados 'aptanaDB', consulte a próxima exceção para obter detalhes.
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver30.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:233)
	at com.aptana.ide.core.db.AptanaDB.getConnection(AptanaDB.java:225)
	at com.aptana.ide.core.db.AptanaDB.execute(AptanaDB.java:97)
	at com.aptana.ide.core.db.FeatureTracker.addFeature(FeatureTracker.java:140)
	at com.aptana.ide.core.db.FeatureTracker.addFeature(FeatureTracker.java:117)
	at com.aptana.ide.core.ui.ping.PingStartup.addFeatureList(PingStartup.java:124)
	at com.aptana.ide.core.ui.ping.PingStartup.updateAnonymousId(PingStartup.java:391)
	at com.aptana.ide.core.ui.ping.PingStartup.access$0(PingStartup.java:367)
	at com.aptana.ide.core.ui.ping.PingStartup$1.run(PingStartup.java:446)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.aptana.ide.core 4 0 2014-09-06 11:36:13.328
!MESSAGE (Build 2.0.5.1278523018) [ERROR] Unable to connect to database
!STACK 0
SQL Exception: Falha ao iniciar o banco de dados 'aptanaDB', consulte a próxima exceção para obter detalhes.
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver30.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:233)
	at com.aptana.ide.core.db.AptanaDB.getConnection(AptanaDB.java:225)
	at com.aptana.ide.core.db.AptanaDB.execute(AptanaDB.java:97)
	at com.aptana.ide.core.db.EventLogger.logEvent(EventLogger.java:142)
	at com.aptana.ide.core.db.EventLogger.logEvent(EventLogger.java:117)
	at com.aptana.ide.core.ui.ping.PingStartup.updateAnonymousId(PingStartup.java:394)
	at com.aptana.ide.core.ui.ping.PingStartup.access$0(PingStartup.java:367)
	at com.aptana.ide.core.ui.ping.PingStartup$1.run(PingStartup.java:446)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.aptana.ide.core 4 0 2014-09-06 11:36:13.417
!MESSAGE (Build 2.0.5.1278523018) [ERROR] Unable to connect to database
!STACK 0
SQL Exception: Falha ao iniciar o banco de dados 'aptanaDB', consulte a próxima exceção para obter detalhes.
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver30.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:233)
	at com.aptana.ide.core.db.AptanaDB.getConnection(AptanaDB.java:225)
	at com.aptana.ide.core.db.AptanaDB.execute(AptanaDB.java:146)
	at com.aptana.ide.core.db.EventLogger.getEvents(EventLogger.java:223)
	at com.aptana.ide.core.ui.ping.PingStartup.updateAnonymousId(PingStartup.java:397)
	at com.aptana.ide.core.ui.ping.PingStartup.access$0(PingStartup.java:367)
	at com.aptana.ide.core.ui.ping.PingStartup$1.run(PingStartup.java:446)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY org.eclipse.ui.workbench 4 0 2014-09-06 11:36:17.336
!MESSAGE Widget disposed too early!
!STACK 0
java.lang.RuntimeException: Widget disposed too early!
	at org.eclipse.ui.internal.WorkbenchPartReference$1.widgetDisposed(WorkbenchPartReference.java:172)
	at org.eclipse.swt.widgets.TypedListener.handleEvent(TypedListener.java:117)
	at org.eclipse.swt.widgets.EventTable.sendEvent(EventTable.java:84)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1176)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1200)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1181)
	at org.eclipse.swt.widgets.Widget.release(Widget.java:1044)
	at org.eclipse.swt.widgets.Control.release(Control.java:3293)
	at org.eclipse.swt.widgets.Composite.releaseChildren(Composite.java:1185)
	at org.eclipse.swt.widgets.Widget.release(Widget.java:1047)
	at org.eclipse.swt.widgets.Control.release(Control.java:3293)
	at org.eclipse.swt.widgets.Composite.releaseChildren(Composite.java:1185)
	at org.eclipse.swt.widgets.Widget.release(Widget.java:1047)
	at org.eclipse.swt.widgets.Control.release(Control.java:3293)
	at org.eclipse.swt.widgets.Composite.releaseChildren(Composite.java:1185)
	at org.eclipse.swt.widgets.Canvas.releaseChildren(Canvas.java:211)
	at org.eclipse.swt.widgets.Decorations.releaseChildren(Decorations.java:467)
	at org.eclipse.swt.widgets.Shell.releaseChildren(Shell.java:2063)
	at org.eclipse.swt.widgets.Widget.release(Widget.java:1047)
	at org.eclipse.swt.widgets.Control.release(Control.java:3293)
	at org.eclipse.swt.widgets.Widget.dispose(Widget.java:446)
	at org.eclipse.swt.widgets.Shell.dispose(Shell.java:1999)
	at org.eclipse.jface.window.Window.close(Window.java:335)
	at org.eclipse.jface.window.ApplicationWindow.close(ApplicationWindow.java:307)
	at org.eclipse.ui.internal.WorkbenchWindow.hardClose(WorkbenchWindow.java:1714)
	at org.eclipse.ui.internal.WorkbenchWindow.busyClose(WorkbenchWindow.java:724)
	at org.eclipse.ui.internal.WorkbenchWindow.access$0(WorkbenchWindow.java:700)
	at org.eclipse.ui.internal.WorkbenchWindow$5.run(WorkbenchWindow.java:816)
	at org.eclipse.swt.custom.BusyIndicator.showWhile(BusyIndicator.java:70)
	at org.eclipse.ui.internal.WorkbenchWindow.close(WorkbenchWindow.java:814)
	at org.eclipse.jface.window.WindowManager.close(WindowManager.java:109)
	at org.eclipse.ui.internal.Workbench$16.run(Workbench.java:930)
	at org.eclipse.core.runtime.SafeRunner.run(SafeRunner.java:42)
	at org.eclipse.ui.internal.Workbench.busyClose(Workbench.java:927)
	at org.eclipse.ui.internal.Workbench.access$15(Workbench.java:856)
	at org.eclipse.ui.internal.Workbench$23.run(Workbench.java:1100)
	at org.eclipse.swt.custom.BusyIndicator.showWhile(BusyIndicator.java:70)
	at org.eclipse.ui.internal.Workbench.close(Workbench.java:1098)
	at org.eclipse.ui.internal.Workbench.close(Workbench.java:1070)
	at org.eclipse.ui.internal.WorkbenchWindow.busyClose(WorkbenchWindow.java:721)
	at org.eclipse.ui.internal.WorkbenchWindow.access$0(WorkbenchWindow.java:700)
	at org.eclipse.ui.internal.WorkbenchWindow$5.run(WorkbenchWindow.java:816)
	at org.eclipse.swt.custom.BusyIndicator.showWhile(BusyIndicator.java:70)
	at org.eclipse.ui.internal.WorkbenchWindow.close(WorkbenchWindow.java:814)
	at org.eclipse.jface.window.Window.handleShellCloseEvent(Window.java:741)
	at org.eclipse.jface.window.Window$3.shellClosed(Window.java:687)
	at org.eclipse.swt.widgets.TypedListener.handleEvent(TypedListener.java:92)
	at org.eclipse.swt.widgets.EventTable.sendEvent(EventTable.java:84)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1176)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1200)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1185)
	at org.eclipse.swt.widgets.Shell.closeWidget(Shell.java:612)
	at org.eclipse.swt.widgets.Shell.gtk_delete_event(Shell.java:1087)
	at org.eclipse.swt.widgets.Widget.windowProc(Widget.java:1562)
	at org.eclipse.swt.widgets.Control.windowProc(Control.java:4594)
	at org.eclipse.swt.widgets.Display.windowProc(Display.java:4191)
	at org.eclipse.swt.internal.gtk.OS._gtk_main_do_event(Native Method)
	at org.eclipse.swt.internal.gtk.OS.gtk_main_do_event(OS.java:7603)
	at org.eclipse.swt.widgets.Display.eventProc(Display.java:1185)
	at org.eclipse.swt.internal.gtk.OS._g_main_context_iteration(Native Method)
	at org.eclipse.swt.internal.gtk.OS.g_main_context_iteration(OS.java:1860)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3110)
	at org.eclipse.ui.internal.Workbench.runEventLoop(Workbench.java:2405)
	at org.eclipse.ui.internal.Workbench.runUI(Workbench.java:2369)
	at org.eclipse.ui.internal.Workbench.access$4(Workbench.java:2221)
	at org.eclipse.ui.internal.Workbench$5.run(Workbench.java:500)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:332)
	at org.eclipse.ui.internal.Workbench.createAndRunWorkbench(Workbench.java:493)
	at org.eclipse.ui.PlatformUI.createAndRunWorkbench(PlatformUI.java:149)
	at com.aptana.radrails.rcp.IDEApplication.start(IDEApplication.java:121)
	at com.aptana.ide.desktop.integration.Application.start(Application.java:56)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:194)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:110)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:79)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:368)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:179)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:559)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:514)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1311)

!ENTRY org.eclipse.ui.workbench 4 0 2014-09-06 11:36:17.339
!MESSAGE Widget disposed too early!
!STACK 0
java.lang.RuntimeException: Widget disposed too early!
	at org.eclipse.ui.internal.WorkbenchPartReference$1.widgetDisposed(WorkbenchPartReference.java:172)
	at org.eclipse.swt.widgets.TypedListener.handleEvent(TypedListener.java:117)
	at org.eclipse.swt.widgets.EventTable.sendEvent(EventTable.java:84)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1176)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1200)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1181)
	at org.eclipse.swt.widgets.Widget.release(Widget.java:1044)
	at org.eclipse.swt.widgets.Control.release(Control.java:3293)
	at org.eclipse.swt.widgets.Composite.releaseChildren(Composite.java:1185)
	at org.eclipse.swt.widgets.Widget.release(Widget.java:1047)
	at org.eclipse.swt.widgets.Control.release(Control.java:3293)
	at org.eclipse.swt.widgets.Composite.releaseChildren(Composite.java:1185)
	at org.eclipse.swt.widgets.Widget.release(Widget.java:1047)
	at org.eclipse.swt.widgets.Control.release(Control.java:3293)
	at org.eclipse.swt.widgets.Composite.releaseChildren(Composite.java:1185)
	at org.eclipse.swt.widgets.Canvas.releaseChildren(Canvas.java:211)
	at org.eclipse.swt.widgets.Decorations.releaseChildren(Decorations.java:467)
	at org.eclipse.swt.widgets.Shell.releaseChildren(Shell.java:2063)
	at org.eclipse.swt.widgets.Widget.release(Widget.java:1047)
	at org.eclipse.swt.widgets.Control.release(Control.java:3293)
	at org.eclipse.swt.widgets.Widget.dispose(Widget.java:446)
	at org.eclipse.swt.widgets.Shell.dispose(Shell.java:1999)
	at org.eclipse.jface.window.Window.close(Window.java:335)
	at org.eclipse.jface.window.ApplicationWindow.close(ApplicationWindow.java:307)
	at org.eclipse.ui.internal.WorkbenchWindow.hardClose(WorkbenchWindow.java:1714)
	at org.eclipse.ui.internal.WorkbenchWindow.busyClose(WorkbenchWindow.java:724)
	at org.eclipse.ui.internal.WorkbenchWindow.access$0(WorkbenchWindow.java:700)
	at org.eclipse.ui.internal.WorkbenchWindow$5.run(WorkbenchWindow.java:816)
	at org.eclipse.swt.custom.BusyIndicator.showWhile(BusyIndicator.java:70)
	at org.eclipse.ui.internal.WorkbenchWindow.close(WorkbenchWindow.java:814)
	at org.eclipse.jface.window.WindowManager.close(WindowManager.java:109)
	at org.eclipse.ui.internal.Workbench$16.run(Workbench.java:930)
	at org.eclipse.core.runtime.SafeRunner.run(SafeRunner.java:42)
	at org.eclipse.ui.internal.Workbench.busyClose(Workbench.java:927)
	at org.eclipse.ui.internal.Workbench.access$15(Workbench.java:856)
	at org.eclipse.ui.internal.Workbench$23.run(Workbench.java:1100)
	at org.eclipse.swt.custom.BusyIndicator.showWhile(BusyIndicator.java:70)
	at org.eclipse.ui.internal.Workbench.close(Workbench.java:1098)
	at org.eclipse.ui.internal.Workbench.close(Workbench.java:1070)
	at org.eclipse.ui.internal.WorkbenchWindow.busyClose(WorkbenchWindow.java:721)
	at org.eclipse.ui.internal.WorkbenchWindow.access$0(WorkbenchWindow.java:700)
	at org.eclipse.ui.internal.WorkbenchWindow$5.run(WorkbenchWindow.java:816)
	at org.eclipse.swt.custom.BusyIndicator.showWhile(BusyIndicator.java:70)
	at org.eclipse.ui.internal.WorkbenchWindow.close(WorkbenchWindow.java:814)
	at org.eclipse.jface.window.Window.handleShellCloseEvent(Window.java:741)
	at org.eclipse.jface.window.Window$3.shellClosed(Window.java:687)
	at org.eclipse.swt.widgets.TypedListener.handleEvent(TypedListener.java:92)
	at org.eclipse.swt.widgets.EventTable.sendEvent(EventTable.java:84)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1176)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1200)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1185)
	at org.eclipse.swt.widgets.Shell.closeWidget(Shell.java:612)
	at org.eclipse.swt.widgets.Shell.gtk_delete_event(Shell.java:1087)
	at org.eclipse.swt.widgets.Widget.windowProc(Widget.java:1562)
	at org.eclipse.swt.widgets.Control.windowProc(Control.java:4594)
	at org.eclipse.swt.widgets.Display.windowProc(Display.java:4191)
	at org.eclipse.swt.internal.gtk.OS._gtk_main_do_event(Native Method)
	at org.eclipse.swt.internal.gtk.OS.gtk_main_do_event(OS.java:7603)
	at org.eclipse.swt.widgets.Display.eventProc(Display.java:1185)
	at org.eclipse.swt.internal.gtk.OS._g_main_context_iteration(Native Method)
	at org.eclipse.swt.internal.gtk.OS.g_main_context_iteration(OS.java:1860)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3110)
	at org.eclipse.ui.internal.Workbench.runEventLoop(Workbench.java:2405)
	at org.eclipse.ui.internal.Workbench.runUI(Workbench.java:2369)
	at org.eclipse.ui.internal.Workbench.access$4(Workbench.java:2221)
	at org.eclipse.ui.internal.Workbench$5.run(Workbench.java:500)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:332)
	at org.eclipse.ui.internal.Workbench.createAndRunWorkbench(Workbench.java:493)
	at org.eclipse.ui.PlatformUI.createAndRunWorkbench(PlatformUI.java:149)
	at com.aptana.radrails.rcp.IDEApplication.start(IDEApplication.java:121)
	at com.aptana.ide.desktop.integration.Application.start(Application.java:56)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:194)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:110)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:79)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:368)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:179)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:559)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:514)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1311)

!ENTRY org.eclipse.ui.workbench 4 2 2014-09-06 11:36:17.425
!MESSAGE Problems occurred when invoking code from plug-in: "org.eclipse.ui.workbench".
!STACK 0
java.lang.NullPointerException
	at org.eclipse.ui.internal.presentations.PresentablePart.setVisible(PresentablePart.java:180)
	at org.eclipse.ui.internal.presentations.util.TabbedStackPresentation.setVisible(TabbedStackPresentation.java:381)
	at org.eclipse.ui.internal.PartStack.setVisible(PartStack.java:1058)
	at org.eclipse.ui.internal.PartSashContainer.remove(PartSashContainer.java:739)
	at org.eclipse.ui.internal.EditorSashContainer.removeEditor(EditorSashContainer.java:295)
	at org.eclipse.ui.internal.EditorAreaHelper.closeEditor(EditorAreaHelper.java:84)
	at org.eclipse.ui.internal.EditorAreaHelper.closeEditor(EditorAreaHelper.java:62)
	at org.eclipse.ui.internal.WorkbenchPage.closeEditors(WorkbenchPage.java:1375)
	at org.eclipse.ui.internal.WorkbenchPage.closeAllEditors(WorkbenchPage.java:1184)
	at org.eclipse.ui.internal.WorkbenchPage.dispose(WorkbenchPage.java:1764)
	at org.eclipse.ui.internal.WorkbenchWindow.closeAllPages(WorkbenchWindow.java:851)
	at org.eclipse.ui.internal.WorkbenchWindow.hardClose(WorkbenchWindow.java:1675)
	at org.eclipse.ui.internal.WorkbenchWindow.busyClose(WorkbenchWindow.java:724)
	at org.eclipse.ui.internal.WorkbenchWindow.access$0(WorkbenchWindow.java:700)
	at org.eclipse.ui.internal.WorkbenchWindow$5.run(WorkbenchWindow.java:816)
	at org.eclipse.swt.custom.BusyIndicator.showWhile(BusyIndicator.java:70)
	at org.eclipse.ui.internal.WorkbenchWindow.close(WorkbenchWindow.java:814)
	at org.eclipse.jface.window.WindowManager.close(WindowManager.java:109)
	at org.eclipse.ui.internal.Workbench$16.run(Workbench.java:930)
	at org.eclipse.core.runtime.SafeRunner.run(SafeRunner.java:42)
	at org.eclipse.ui.internal.Workbench.busyClose(Workbench.java:927)
	at org.eclipse.ui.internal.Workbench.access$15(Workbench.java:856)
	at org.eclipse.ui.internal.Workbench$23.run(Workbench.java:1100)
	at org.eclipse.swt.custom.BusyIndicator.showWhile(BusyIndicator.java:70)
	at org.eclipse.ui.internal.Workbench.close(Workbench.java:1098)
	at org.eclipse.ui.internal.Workbench.close(Workbench.java:1070)
	at org.eclipse.ui.internal.WorkbenchWindow.busyClose(WorkbenchWindow.java:721)
	at org.eclipse.ui.internal.WorkbenchWindow.access$0(WorkbenchWindow.java:700)
	at org.eclipse.ui.internal.WorkbenchWindow$5.run(WorkbenchWindow.java:816)
	at org.eclipse.swt.custom.BusyIndicator.showWhile(BusyIndicator.java:70)
	at org.eclipse.ui.internal.WorkbenchWindow.close(WorkbenchWindow.java:814)
	at org.eclipse.jface.window.Window.handleShellCloseEvent(Window.java:741)
	at org.eclipse.jface.window.Window$3.shellClosed(Window.java:687)
	at org.eclipse.swt.widgets.TypedListener.handleEvent(TypedListener.java:92)
	at org.eclipse.swt.widgets.EventTable.sendEvent(EventTable.java:84)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1176)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1200)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1185)
	at org.eclipse.swt.widgets.Shell.closeWidget(Shell.java:612)
	at org.eclipse.swt.widgets.Shell.gtk_delete_event(Shell.java:1087)
	at org.eclipse.swt.widgets.Widget.windowProc(Widget.java:1562)
	at org.eclipse.swt.widgets.Control.windowProc(Control.java:4594)
	at org.eclipse.swt.widgets.Display.windowProc(Display.java:4191)
	at org.eclipse.swt.internal.gtk.OS._gtk_main_do_event(Native Method)
	at org.eclipse.swt.internal.gtk.OS.gtk_main_do_event(OS.java:7603)
	at org.eclipse.swt.widgets.Display.eventProc(Display.java:1185)
	at org.eclipse.swt.internal.gtk.OS._g_main_context_iteration(Native Method)
	at org.eclipse.swt.internal.gtk.OS.g_main_context_iteration(OS.java:1860)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3110)
	at org.eclipse.ui.internal.Workbench.runEventLoop(Workbench.java:2405)
	at org.eclipse.ui.internal.Workbench.runUI(Workbench.java:2369)
	at org.eclipse.ui.internal.Workbench.access$4(Workbench.java:2221)
	at org.eclipse.ui.internal.Workbench$5.run(Workbench.java:500)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:332)
	at org.eclipse.ui.internal.Workbench.createAndRunWorkbench(Workbench.java:493)
	at org.eclipse.ui.PlatformUI.createAndRunWorkbench(PlatformUI.java:149)
	at com.aptana.radrails.rcp.IDEApplication.start(IDEApplication.java:121)
	at com.aptana.ide.desktop.integration.Application.start(Application.java:56)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:194)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:110)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:79)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:368)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:179)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:559)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:514)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1311)

!ENTRY com.aptana.ide.core 4 0 2014-09-06 11:36:18.101
!MESSAGE (Build 2.0.5.1278523018) [ERROR] An error occurred while shutting down the database
!STACK 0
java.sql.SQLException: No suitable driver found for jdbc:derby:aptanaDB;shutdown=true
	at java.sql.DriverManager.getConnection(DriverManager.java:596)
	at java.sql.DriverManager.getConnection(DriverManager.java:233)
	at com.aptana.ide.core.db.AptanaDB.shutdown(AptanaDB.java:320)
	at com.aptana.ide.core.AptanaCorePlugin.stop(AptanaCorePlugin.java:73)
	at org.eclipse.osgi.framework.internal.core.BundleContextImpl$2.run(BundleContextImpl.java:843)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.eclipse.osgi.framework.internal.core.BundleContextImpl.stop(BundleContextImpl.java:836)
	at org.eclipse.osgi.framework.internal.core.BundleHost.stopWorker(BundleHost.java:474)
	at org.eclipse.osgi.framework.internal.core.AbstractBundle.suspend(AbstractBundle.java:546)
	at org.eclipse.osgi.framework.internal.core.Framework.suspendBundle(Framework.java:1098)
	at org.eclipse.osgi.framework.internal.core.StartLevelManager.decFWSL(StartLevelManager.java:593)
	at org.eclipse.osgi.framework.internal.core.StartLevelManager.doSetStartLevel(StartLevelManager.java:261)
	at org.eclipse.osgi.framework.internal.core.StartLevelManager.shutdown(StartLevelManager.java:216)
	at org.eclipse.osgi.framework.internal.core.InternalSystemBundle.suspend(InternalSystemBundle.java:266)
	at org.eclipse.osgi.framework.internal.core.Framework.shutdown(Framework.java:685)
	at org.eclipse.osgi.framework.internal.core.Framework.close(Framework.java:583)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.shutdown(EclipseStarter.java:409)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:200)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:559)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:514)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1311)
